conf/trust.routes

```diff
@@ -16,3 +16,6 @@ GET         /:journeyId/no-chrn                            uk.gov.hmrc.minorenti
 
 GET         /:journeyId/check-your-answers-business      uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CheckYourAnswersController.show(journeyId)
 POST        /:journeyId/check-your-answers-business      uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CheckYourAnswersController.submit(journeyId)
+
+GET         /:journeyId/cannot-confirm-business            uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.errorControllers.CannotConfirmBusinessController.show(journeyId)
+POST        /:journeyId/cannot-confirm-business            uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.errorControllers.CannotConfirmBusinessController.submit(journeyId: String)
```

Comment:
I suspect journeyId should be declared as a String for most of the routes.

Reply:
I think it defaults to string which is why most of the routes in our services don't specify it